### PR TITLE
[DNM] Split bindings

### PR DESCRIPTION
**Pull request by rhlsthrm**

### CODE REVIEW

File: packages/agents/cartographer/poller/package.json [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
   "files": [
     "dist/**/*",
     "src/**/*"
   ],
   "scripts": {
-    "dev": "concurrently -i -k -n 'app,tsc' -c 'green,cyan' 'nodemon'",
-    "start": "pm2 start dist/index.js --source-map-support --kill-timeout 5000 --cron-restart=\"* * * * *\" --time",
-    "stop": "pm2 delete index",
+    "dev": "concurrently -i -k -n 'app,tsc' -c 'green,cyan' 'pm2-runtime --watch cartographer.config.js'",
+    "start": "pm2-runtime cartographer.config.js",
```
Comment (preethamr) [inline packages/agents/cartographer/poller/package.json:12]: We need to update the Dockerfile `CMD` accordingly. I can do that update.

Comment (rhlsthrm) [reply packages/agents/cartographer/poller/package.json:12]: Yes, we will need to modify Dockerfile. Eventually, we can split into separate Docker images for each pm2 app.

Comment (preethamr) [reply packages/agents/cartographer/poller/package.json:12]: Updated the Dockerfile.

Right, that would align with any auto-scaling we would use, and can scale them seperately

**Comment by preethamr** [comment]
LGTM. And tests passing.

**Pull request merged**